ambry-rest/src/main/java/com.github.ambry.rest/NettyResponseChannel.java

```diff
@@ -60,6 +62,10 @@
  * will be accepted and all scheduled writes will be notified of the failure.
  */
 class NettyResponseChannel implements RestResponseChannel {
+  // add to this list if the connection needs to be closed on certain errors on GET, DELETE and HEAD.
+  // for a POST, we always close the connection on error because we expect the channel to be in a bad state.
+  static final List<HttpResponseStatus> CLOSE_CONNECTION_ERROR_STATUSES = new ArrayList<>();
```

Comment:
Why do you want to add something which is never used for now. If we have anything response status that we know off, we can have this list. If not, I don't see a point having it, just because in future we might add something to the list. It creates confusion.